components/webhooks/cmd/worker.go

```diff
@@ -1,35 +1,66 @@
 package cmd
 
 import (
+	"net/http"
+	"strings"
 	"syscall"
+	"time"
 
+	"github.com/formancehq/stack/libs/go-libs/httpserver"
 	"github.com/formancehq/stack/libs/go-libs/logging"
 	"github.com/formancehq/stack/libs/go-libs/service"
 	"github.com/formancehq/webhooks/cmd/flag"
 	"github.com/formancehq/webhooks/pkg/otlp"
+	"github.com/formancehq/webhooks/pkg/storage/postgres"
 	"github.com/formancehq/webhooks/pkg/worker"
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	"go.uber.org/fx"
 )
 
-var workerCmd = &cobra.Command{
-	Use:   "worker",
-	Short: "Run webhooks worker",
-	RunE:  RunWorker,
+func newWorkerCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "worker",
+		Short: "Run webhooks worker",
+		RunE:  runWorker,
+	}
 }
 
-func RunWorker(cmd *cobra.Command, _ []string) error {
+func runWorker(cmd *cobra.Command, _ []string) error {
 	logging.FromContext(cmd.Context()).Debugf(
-		"starting webhooks worker module: env variables: %+v viper keys: %+v",
-		syscall.Environ(), viper.AllKeys())
+		"starting webhooks worker module: env variables: %+v viper settings: %+v",
+		syscall.Environ(), viper.AllSettings())
+
+	retriesSchedule := make([]time.Duration, 0)
+	rs := viper.GetString(flag.RetriesSchedule)
```

Comment:
Why not used a StringSlice instead of manually reimplement what viper already do?